config/initializers/cors.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "*"
+    resource "/api/v1/*",
+             headers: :any,
+             methods: %i[ get post put patch delete options head ]
+  end
+
+  allow do
+    origins "localhost:3000", "https://transientbug.ninja"
+    resource "/*",
+             headers: :any,
+             methods: %i[ get post put patch delete options head ]
+  end
```

Comment:
Is the default to deny everything? Or should you be specifying that?

Reply:
Same origin allow all, I think is the default, but I'm not 100% sure